src/transform/class/PotentialMethod.js

```diff
@@ -70,4 +78,31 @@ class PotentialMethod {
       replacements: [],
     });
   }
+
+  modifySuperCalls() {
```

Comment:
I think it would be better to have a method like `getBody()` which would return `this.methodNode.body`, which in the case of constructor and presence of superclass would have the super() calls transformed.